fastlane/lib/fastlane/actions/appaloosa.rb

```diff
@@ -104,6 +104,7 @@ def self.upload_on_appaloosa(api_key, store_id, binary_path, screenshots, group_
         req.body = { store_id: store_id,
                      api_key: api_key,
                      mobile_application_update: {
+                       description: description,
```

Comment:
So if it is optional, maybe we should only insert the `description` key into the dictionary if there is a description present to fill the value. I would be a little worried to pass `nil` and if that would wipe out some data that may already be setup in Appaloosa, or just to not add unnecessary parameters.